src/cpp/multisolver_interface/SolverCbc.cpp

```diff
@@ -95,8 +95,8 @@ void SolverCbc::free()
 void SolverCbc::write_prob_mps(const std::filesystem::path& filename)
 {
     const int numcols = get_ncols();
-    std::shared_ptr<char[]> shared_integrality(new char[numcols]);
-    char* integrality = shared_integrality.get();
+    std::vector<char> shared_integrality(static_cast<std::size_t>(numcols));
+    char* integrality = shared_integrality.data();
```

Comment:
do we need to define : 
```c++
char* integrality = shared_integrality.data();
```
?
Or we could use **shared_integrality** instead ?

Reply:
C APIs don't know about `std::vector<char>`